plutus-core/plutus-ir/test/transform/inline/letAppMulti

```diff
@@ -0,0 +1,42 @@
+{- 
+Multiple occurrences of a let binding that is a function application (funApp). It should not get 
+inlined by the unconditional inlining optimization. The optimization that inlines fully
+applied function should also not inline funApp, even though it is fully applied.
+Because it is an application, so it's not acceptable in terms of both cost and size. This test
+verifies that the cost and size heuristics are checked.
+
+let 
+  appNum :: Integer
+  appNum = 4
+  in 
+    let funApp :: Integer -> Integer
+        funApp x = addInteger appNum x
+    in
+      let k :: Integer -> Integer
+      k = funApp
+in funApp appNum
```

Comment:
This still won't (and shouldn't) get inlined, because replacing `funApp appNum` with `addInteger appNum x` _increases_ the size of the program.

Reply:
Oh, I just realized that that is exactly what the test is for, so disregard the comment above.